fnet/data/multichtiffdataset.py

```diff
@@ -0,0 +1,62 @@
+import skimage.external.tifffile as tifffile
+import pandas as pd
+import numpy as np
+import torch
+
+from fnet.data.fnetdataset import FnetDataset
+import fnet.transforms as transforms
+
+
+class MultiChTiffDataset(FnetDataset):
+    """Dataset for multi-channel tiff files.
+
+    Currently assumes that images are loaded in ZCXY format
+
+    """
+
+    def __init__(self, dataframe: pd.DataFrame = None, path_csv: str = None,
+                 transform_signal=[transforms.normalize],
+                 transform_target=None):
```

Comment:
Shouldn't the target also be normalized? As defaults, I think either both signal and target should be normalized or nothing should be done at all.